modules/carto/src/api/parseMap.ts

```diff
@@ -254,11 +254,12 @@ function createChannelProps(
     const maxIconSize = getMaxMarkerSize(visConfig, visualChannels);
     const {getPointRadius, getFillColor} = result;
     result.pointType = 'icon';
+    const useMaskedIcons = visConfig.filled;
```

Comment:
How about just destructuring here both required pieces of info...

```suggestion
    const { filled: useMaskedIcons, customMarkersUrl: fallbackUrl } = visConfig;
```

Reply:
Suggestion applied, thanks